webrender/src/clip_scroll_node.rs

```diff
@@ -80,7 +90,7 @@ pub enum NodeType {
 impl NodeType {
     fn is_reference_frame(&self) -> bool {
         match *self {
-            NodeType::ReferenceFrame(_) => true,
+            NodeType::Positioning { kind: PositioningNodeType::ReferenceFrame(_), .. } => true,
```

Comment:
do we need all those `..` sprinkled in `Positioning` matches?

Reply:
Yep - if all goes according to plan, I'll need to add one common field to the Spatial node type, which will be used as an index / identifier into a (typically much smaller) array of transforms that we now need to send to the GPU.